pkgs/swiftgen/swift2objc/lib/src/parser/_core/utils.dart

```diff
@@ -16,8 +16,19 @@ Json readJsonFile(String jsonFilePath) {
   return Json(jsonDecode(jsonStr));
 }
 
+const idDelim = "-";
+
+extension AddIdSuffix on String {
+  String addIdSuffix(String suffix) => "$this$idDelim$suffix";
+}
+
 String parseSymbolId(Json symbolJson) {
-  return symbolJson["identifier"]["precise"].get();
+  final idJson = symbolJson["identifier"]["precise"];
+  final String id = idJson.get();
+  if (id.contains(idDelim)) {
```

Comment:
`assert(!id.contains(idDelim), 'Symbold id $id at path ${idJson.path} contains $idDelim which is not expected')`

Reply:
done